### DISCUSSION TITLE
What is the optimal configuration for writing with wacom pencils?

### DISCUSSION THREAD

**Discussion by vladQll** [flxzt/rnote#290] [Q&A]
OS = fedora 36
tableta digitalizadora =  wacom intuos S ctl-4100

Hello, I have a question is that I want to use rnote to take notes, but I have problems with the strokes when writing my handwriting is good with pencil and paper, but in rnote is not happening because when I try to write in the most natural way possible, the strokes are drawn wrong to make curves, an example is when making, parentheses, or characters like the letters 'd', 'a', 'o', then you should configure something in the settings of the brushes? which would be the optimal configuration.

_image about the problem on rnote_  
![imagen](https://user-images.githubusercontent.com/28522011/194964947-54bf3a13-de3a-4e31-91c7-56b7198c58fc.png)

_image as I wish I could write_
![imagen](https://user-images.githubusercontent.com/28522011/194965740-8cf0370b-755c-4e27-b1f7-863954668e3b.png)

**Comment by flxzt** [comment]
Hi. Try the newest beta, it features a new stroke modeler which should improve handwriting significantly, with results comparable to xournalpp's smoothing algorithms.
You can also try to pick another pressure curve. Try playing around with that, for example the `Parabola` curves result in a thinner stroke which is less sensitive to pressure input.

![Bildschirmfoto vom 2022-10-19 12-52-12](https://user-images.githubusercontent.com/19841886/196671808-71aaea95-06ea-4acc-bd80-2c7ed0d533dc.png)

**Reply by vladQll** [reply]
hello would you be so kind to share with me your optimal settings that you use to write with your tableta on rnote, I have tried updating rnote from the terminal and yes there is a noticeable change but I would like to know if there are any special settings, I do not understand much about things like curve, smoothing etc, and my native language is not English and I do not understand what it means or what certain options of the pen settings do.
